ui/src/notifications/Notifications.tsx

```diff
@@ -161,20 +188,34 @@ export default function Notifications({
       <div className="flex flex-row justify-end pt-2">
         {isMobile && hasUnreads && MarkAsRead}
       </div>
-      {notifications.map((grouping) => (
-        <div key={grouping.date}>
-          <h2 className="my-4 text-lg font-bold text-gray-400">
-            {grouping.date}
-          </h2>
-          <ul className="space-y-2">
-            {grouping.bins.map((b) => (
-              <li key={b.time}>
-                <Notification bin={b} />
-              </li>
-            ))}
-          </ul>
-        </div>
-      ))}
+      {loaded ? (
+        notifications.length === 0 ? (
+          <div className="flex w-full items-center justify-center">
+            <span className="text-xl font-semibold">
+              No notifications for {group ? 'this' : 'any'} group.
```

Comment:
Perfect, we'll fold in the root-level first-run wayfinding messages here. We can also do something group-specific.

cc @lukestiles